report_pre_thesis/src/sections/methodology.tex

```diff
@@ -98,6 +98,54 @@ \subsubsection{Outlier Removal with Mahalanobis Distance and Chi-Squared Test}\l
 It should be noted that our outlier removal process is conservative to avoid removing too many samples and as such, we have set our significance level $\alpha$ to 0.975.
```

Comment:
Is 0.975 still true? Did we change it to 0.95?

Reply:
Remained at 0.975 throughout